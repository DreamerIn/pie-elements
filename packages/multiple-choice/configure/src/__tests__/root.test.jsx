describe('firstAvailableIndex', () => {
  it('finds first free index ', () => {});
});
