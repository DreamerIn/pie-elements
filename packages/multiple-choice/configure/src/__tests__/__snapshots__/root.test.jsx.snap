// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root snapshot renders 1`] = `
<WithStyles(Main)
  model={
    Object {
      "choiceMode": "checkbox",
      "choicePrefix": "numbers",
      "choices": Array [
        Object {
          "correct": true,
          "feedback": Object {
            "type": "none",
            "value": "",
          },
          "label": "Sweden",
          "value": "sweden",
        },
        Object {
          "feedback": Object {
            "type": "none",
            "value": "",
          },
          "label": "Iceland",
          "value": "iceland",
        },
        Object {
          "feedback": Object {
            "type": "none",
            "value": "",
          },
          "label": "Norway",
          "value": "norway",
        },
        Object {
          "correct": true,
          "feedback": Object {
            "type": "none",
            "value": "",
          },
          "label": "Finland",
          "value": "finland",
        },
      ],
      "configure": Object {},
      "itemStem": "Which of these northern European countries are EU members?",
      "partialScoring": false,
    }
  }
  onAddChoice={[Function]}
  onChoiceChanged={[Function]}
  onChoiceModeChanged={[Function]}
  onChoicePrefixChanged={[Function]}
  onLockChoiceOrderChanged={[Function]}
  onPartialScoringChanged={[Function]}
  onPromptChanged={[Function]}
  onRemoveChoice={[Function]}
/>
`;
